includes/templates/single-web-story.php

```diff
@@ -83,40 +83,8 @@
 </head>
 <body>
 <?php
-
-$metadata = Story_Post_Type::get_schemaorg_metadata();
-
-if ( isset( $metadata['publisher']['logo']['url'] ) ) {
-	$publisher_logo_src = $metadata['publisher']['logo']['url'];
-} elseif ( isset( $metadata['publisher']['logo'] ) && is_string( $metadata['publisher']['logo'] ) ) {
-	$publisher_logo_src = $metadata['publisher']['logo'];
-} else {
-	$publisher_logo_src = admin_url( 'images/wordpress-logo.png' );
-}
-
-$publisher = isset( $metadata['publisher']['name'] ) ? $metadata['publisher']['name'] : get_option( 'blogname' );
-
-$meta_images = Media::get_story_meta_images();
-
-$poster_portrait = isset( $meta_images['poster-portrait'] ) ? $meta_images['poster-portrait'] : plugins_url( 'assets/images/fallback-poster.jpg', WEBSTORIES_PLUGIN_FILE );
-
+// @todo Filter content in PHP to ensure AMP validity and to allow changing publisher logo etc.
```

Comment:
I'm not sure I understand this? We want the rendering in JS, but the PHP to post-process it anyway?

Reply:
Ah, just saw [this comment](https://github.com/google/web-stories-wp/issues/233#issuecomment-588149756) now. Got it!